src/pykeen/models/base.py

```diff
@@ -523,6 +529,19 @@ def score_t_inverse(self, hr_batch: torch.LongTensor, slice_size: Optional[int]
         r_inv_h = self._prepare_inverse_batch(batch=hr_batch, index_relation=1)
         return self.score_h(rt_batch=r_inv_h, slice_size=slice_size)
 
+    def score_r_inverse(self, ht_batch: torch.LongTensor, slice_size: Optional[int] = None):
+        """Score all rels for a batch of (h,t)-pairs using the rel predictions for the inverses $(t,*_{inv},h)$."""
```

Comment:
What is described in the docstring does make sense from my point of view: We compute scores for all relations, but this time, using the inverse representation and flipped head & tail.

```
score[i] = score(t, r_i_inv, h)
```

implementing seems to depend on fixing what `score_r` computes, which currently is equal to scores for normal *and* inverse relations (cf.  #752 )